conf/st2.dev.conf

```diff
@@ -5,7 +5,6 @@
 host = 0.0.0.0
 port = 9101
 logging = st2api/conf/logging.conf
-serve_webui_files = True
 # allow_origin is required for handling CORS in st2 web UI.
 # allow_origin = http://myhost1.example.com:3000,http://myhost2.example.com:3000
```

Comment:
CORS is another problem I haven't solved yet. Seems like the only way is to set it to * in st2_deploy and call it a day. On production, this thing would be handled by nginx anyway and I'm starting to have some doubts it's useful in the first place (we need to consult with security. For now, I can picture a couple of ways how to neutralize CORS during phishing attempt).